tools/mk-generated.sh

```diff
@@ -26,16 +26,23 @@ set -e
 	echo "Current branch resolved to: $current_branch"
 	gen_branch="__generated-$current_branch"
 	repo_root=$(git rev-parse --show-toplevel)
-	cd "$repo_root" && ./gradlew :aws:sdk:assemble
 	target="$(mktemp -d)"
+  # Generate and copy AWS SDK models
```

Comment:
something is messy in this file, I think it ended up mixed tabs and spaces somehow, do you mind just changing the file to spaces

Reply:
Yep, I'll retab it.